intuita-webview/src/codemodList/CodemodArguments/index.tsx

```diff
@@ -45,18 +45,14 @@ const CodemodArguments = ({
 	rootPath,
 	executionPath,
 }: Props) => {
-	const onChangeFormField =
-		(fieldName: string) => (e: Event | React.FormEvent<HTMLElement>) => {
-			const value = (e as React.ChangeEvent<HTMLInputElement>).target
-				.value;
-
-			vscode.postMessage({
-				kind: 'webview.global.setCodemodArgument',
-				hashDigest,
-				name: fieldName,
-				value,
-			});
-		};
+	const onChangeFormField = (fieldName: string) => (value: string) => {
```

Comment:
do you think we should type `fieldName` as a type literal to ensure proper names?

Reply:
we already have type literal for `kind`, `fieldName` is just a user-defined argument name